bundles/org.openhab.binding.opensprinkler/src/main/feature/feature.xml

```diff
@@ -3,6 +3,7 @@
 	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
 	<feature name="openhab-binding-opensprinkler" description="OpenSprinkler Binding" version="${project.version}">
 		<feature>openhab-runtime-base</feature>
+		<feature dependency="true">javax.measure.unit-api</feature>
```

Comment:
Why is it necessary to add this dependency ?

Reply:
Removed and it works fine without.